pkg/packagekit/updates.jsx

```diff
@@ -265,9 +265,19 @@ class UpdateItem extends React.Component {
 
         var type;
         if (info.severity === PK_INFO_ENUM_SECURITY) {
+            var classes = "pficon pficon-security";
+
+            // parse Red Hat security update classification from vendor_urls
+            var secSeverity = info.vendor_urls.find(value => value.startsWith("https://access.redhat.com/security/updates/classification/#"));
```

Comment:
This will return the first classification.  Shouldn't we check them all and use the most severe one?

Reply:
That would be odd (package updates are supposed to have one classification, not multiple contradictory ones), but I can do that. It means introducing a hardcoded list of severities, but I suppose they won't change that often :-)

Reply:
> package updates are supposed to have one classification, not multiple contradictory ones

Okay, then I hereby retract my comment. :)